app/src/main/kotlin/com/wire/android/ui/settings/devices/SelfDevicesScreen.kt

```diff
@@ -71,43 +71,47 @@ fun SelfDevicesScreenContent(
                 modifier = Modifier
                     .padding(paddingValues)
                     .fillMaxSize()
-                    .background(color = MaterialTheme.wireColorScheme.surface)
             ) {
                 when (state.isLoadingClientsList) {
                     true -> items(count = 4, itemContent = { Device() })
                     false -> {
                         state.currentDevice?.let { currentDevice ->
-                            item {
-                                FolderHeader(
-                                    name = context.getString(R.string.current_device_label).uppercase(),
-                                    modifier = Modifier
-                                        .fillMaxWidth()
-                                        .background(color = headerColor)
-                                )
-                                DeviceItem(currentDevice, placeholder = false)
-                                Divider()
-                            }
-                        }
-                        item {
-                            FolderHeader(
-                                name = context.getString(R.string.other_devices_label).uppercase(),
-                                modifier = Modifier
-                                    .fillMaxWidth()
-                                    .background(color = headerColor)
+                            folderWithElements(
+                                context.getString(R.string.current_device_label),
+                                listOf(currentDevice)
                             )
-                            Divider()
-                        }
-                        itemsIndexed(state.deviceList) { index, item ->
-                            DeviceItem(item, placeholder = false)
-                            if (index < state.deviceList.lastIndex) Divider()
                         }
+                        folderWithElements(
+                            context.getString(R.string.other_devices_label),
+                            state.deviceList
+                        )
                     }
                 }
             }
         },
     )
 }
 
+private fun LazyListScope.folderWithElements(
```

Comment:
Since this is a Devices specific option, could we rename to:
```suggestion
private fun LazyListScope.folderWithDeviceItems(
```

Reply:
Good idea 👍